dashboard/app/models/lesson.rb

```diff
@@ -171,6 +175,11 @@ def spelling_bee?
     script_levels.first.oldest_active_level.spelling_bee?
   end
 
+  # We number lessons that either have lesson plans or are not lockable
+  def numbered_lesson?
+    has_lesson_plan || !lockable
```

Comment:
I guess if we are going to be comparing the output of this using == to another boolean, then this should start with `!!has_lesson_plan`